### PR TITLE
Add KTeaTime

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://invent.kde.org/utilities/kteatime/-/issues/1
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://invent.kde.org/utilities/kteatime/-/commit/bb8bd750080380d8b230244e9d2e52772a4b33ac https://invent.kde.org/utilities/kteatime/-/merge_requests/7

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build org.kde.kteatime

**Comment by Mailaender** [comment]
bot, build org.kde.kteatime

**Comment by hfiguiere** [comment]
cc @flathub/kde

File: org.kde.kteatime.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.kde.org/stable/release-service/22.04.1/src/kteatime-22.04.1.tar.xz",
+          "sha256": "d5bf75bdc3cafe4b7dc9b4ddff9baacf0f7f903d2a97fa45abf7739b9588655b",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": 8763,
+            "stable-only": true,
+            "url-template": "https://download.kde.org/stable/kteatime/$version/kteatime-$version.tar.xz"
```
Comment (tsdgeos) [inline org.kde.kteatime.json:28]: wrong url

**Comment by Mailaender** [comment]
bot, build org.kde.kteatime

**Comment by travier** [comment]
LGTM. Works as expected.

**Pull request closed without merging**